.github/workflows/php-release.yml

```diff
@@ -0,0 +1,20 @@
+name: PHP Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  packagist:
+    name: Update Packagist
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Update Packagist
+        run: |
+          curl -XPOST "https://packagist.org/api/update-package?username=svix&apiToken=$PACKAGIST_API_KEY" \
```

Comment:
Seriously? They expect you to pass a token as a query param?

Reply:
yep